lib/message/processMIME.js

```diff
@@ -55,7 +55,7 @@ const verifySignature = async ({ publicKeys, date }, data) => {
         signature
     });
 
-    return { subdata, verified };
+    return { subdata, verified, signature };
```

Comment:
`verifyMessage` also returns a `signatures` array similar to `decryptMessage`. I would return that, instead.